ui/src/components/text-truncate/QasTextTruncate.vue

```diff
@@ -185,20 +190,39 @@ const classes = computed(() => [`text-${props.color}`, `text-${defaultTypography
 
 const formattedText = computed(() => props.list.length || props.text ? displayText.value : props.emptyText)
 
+/**
+ * Se estiver em modo contador (list prop preenchida) e não houver descrição
+ * personalizada no diálogo, então mostra o slot de descrição.
+ */
+const showDescriptionSlot = computed(() => isCounterMode.value && !hasDialogDescription?.value)
+
 // composable functions
 function useDialog ({ props, textContent }) {
   // reactive vars
   const show = ref(false)
   const searchModel = ref([])
 
-  // computed
-  const description = computed(() => props.text || textContent.value)
+  // computeds
+  const hasDialogDescription = computed(() => !!props.dialogProps?.description)
+
+  const description = computed(() => {
+    // Se dialogProps.description estiver setado, usa ele
+    if (hasDialogDescription.value) {
+      return props.dialogProps.description
+    }
+
+    // Senão, usa o comportamento padrão
+    return props.text || textContent.value
```

Comment:
`return props.dialogProps?.description || props.text || textContent.value` não funciona? description vai ser undefined caso não for passado não?